ui/src/routes/guides/indicators.svelte

```diff
@@ -0,0 +1,23 @@
+<script>
+	import {_screen} from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
+	import IndicatorsMedium
+		from 'app/components/content/guides/IndicatorsMedium.svelte';
+	import IndicatorsSmall
+	from 'app/components/content/guides/IndicatorsSmall.svelte';
```

Comment:
the `from` line needs indenting:

```suggestion
	import IndicatorsSmall
		from 'app/components/content/guides/IndicatorsSmall.svelte';
```